patches/chrome-browser-resources-settings-settings_icons_css.html.patch

```diff
@@ -0,0 +1,27 @@
+diff --git a/chrome/browser/resources/settings/settings_icons_css.html b/chrome/browser/resources/settings/settings_icons_css.html
+index 7a4e72d47e8e79a89aacd2f84fa47575d122e0a8..5a4c8832a7b91499f19b4d08468639f1dc453383 100644
+--- a/chrome/browser/resources/settings/settings_icons_css.html
++++ b/chrome/browser/resources/settings/settings_icons_css.html
+@@ -4,14 +4,14 @@
+ <dom-module id="settings-icons">
+   <template>
+     <style>
+-<if expr="chromeos">
```

Comment:
I'm confused about this patch. This only applies to chromeos

Reply:
looks like just an indentation thing? I can fix it